lte/gateway/python/magma/enodebd/stats_manager.py

```diff
@@ -126,14 +126,18 @@ def _check_rf_tx_for_handler(self, handler: EnodebAcsStateMachine) -> None:
     def _get_enb_label_from_request(self, request) -> str:
         label = 'default'
         ip = request.headers.get('X-Forwarded-For')
+
         if ip is None:
-            ip = request.remote_addr
+            peername = request.transport.get_extra_info('peername')
+            if peername is not None:
+                ip, _ = peername
 
         if ip is None:
             return label
 
         try:
-            label = self.enb_manager.get_serial(ip)
+            label = self.enb_manager.get_serial_of_ip(ip)
```

Comment:
one minor nit:        get_serial does this  `return self.enb_serial_by_ip[ip]`, we might want to wrap this within a try block and in case of exception return a 'default' label